internal/forwarder/ports.go

```diff
@@ -41,18 +41,17 @@ func splitPort(port string) (local, remote string) {
 func translateServicePortToTargetPort(port string, svc corev1.Service, pod corev1.Pod) (string, error) {
 	localPort, remotePort := splitPort(port)
 
-	// nolint:gosec
-	portnum, err := strconv.Atoi(remotePort)
```

Comment:
Previously, if `remotePort` was a very large numeric string, this code was vulnerable to integer overflow, as identified by the linter. L58 is the vulnerability, where `int32(portnum)` is invoked. If `portnum` has more than 32 bits, the program will panic. By using `ParseInt` and setting the bits to 32, we'll instead get an error on L44.